llvm/test/CodeGen/NVPTX/sad-intrins.ll

```diff
@@ -0,0 +1,50 @@
+; RUN: llc < %s -march=nvptx -mcpu=sm_50 | FileCheck %s
+
+; CHECK-LABEL: test_sad_i16
+; CHECK: sad.s16
```

Comment:
It would be good to check that we pass the arguments correctly.
Here and in [mulhi-intrins.ll](https://github.com/llvm/llvm-project/pull/78385/files#diff-8f0bf66a6d1e6e431bc85fb20aea9b7a5adbeffac5b04e629d36251d1e9dd463) above it may be useful to autogenerate the tests 
https://llvm.org/docs/TestingGuide.html#generating-assertions-in-regression-tests

Reply:
Sounds good, updated both tests with `update_llc_test_checks.py`